fitwsarah_frontend/src/views/AdminPanelPage/AdminCreateCoachNotes.js

```diff
@@ -76,7 +76,7 @@ function AdminCreateCoachNotes() {
             content_FR: newCoachNote.content_FR,
         };
         try {
-            const response = await fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/coachnotes`, {
+            const response = await fetch(`${process.env.REACT_APP_DEVELOPMENT_URL}/api/v1/coachnotes`, {
```

Comment:
Why are you using development URL here ?

Reply:
i just fixed it